### PR TITLE
Add GRASShopper examples to regress tests

### LINKED ISSUE #131: Make travis test GRASShopper benchmarks

### PR DESCRIPTION
**Pull request by MattWindsor91**
Added one fabricated 'failing' GRASShopper test, maybe we need more?

Fixes #131, I think.

### CODE REVIEW

File: Examples/FailGH/ticketLockFlippedIf.cvf [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// Allocatable version of Examples/Pass/ticketLock.cvf.
```
Comment (septract) [inline Examples/FailGH/ticketLockFlippedIf.cvf]: Maybe add a comment saying why this example is incorrect

**Comment by septract** [comment]
Not sure what other tests would be interesting. This is enough to get started anyway.

File: Examples/FailGH/ticketLockFlippedIf.cvf (hunk 1 above)
Comment (MattWindsor91) [reply Examples/FailGH/ticketLockFlippedIf.cvf]: Ok

**Comment by MattWindsor91** [comment]
@septract I've updated the comment ^^

**Comment by septract** [comment]
Great, :shipit:

**Pull request merged**